### PR TITLE
fix(block_scrolling): avoid deprecation message

### PR DESCRIPTION
**Pull request by sufius**
fast solution to remove warnings

### CODE REVIEW

**Comment by poul-kg** [comment]
I want this fix :)

File: .gitignore [hunk 1]
```diff
@@ -7,4 +7,3 @@ cypress/screenshots
 cypress/fixtures
 cypress/plugins
 cypress/support
-dist
```
Comment (JackuB) [inline .gitignore]: I would advise against having `/dist` in public repo in this case. We would need to ensure that it is always up to date with whatever is in master, which would require a precommit hook or something.

File: .npmignore [hunk 2]
```diff
@@ -10,5 +10,3 @@ node_modules
 cypress.json
 index.html
 webpack.config.js
-dist/ace-diff-light.min.js
```
Comment (JackuB) [inline .npmignore]: These `.min.js` files are just artifacts from the webpack pipeline and shouldn't be published.

File: src/index.js [hunk 3]
```diff
@@ -115,6 +115,8 @@ function AceDiff(options) {
   this.editors.right.ace.setReadOnly(!this.options.right.editable);
   this.editors.left.ace.setTheme(getTheme(this, C.EDITOR_LEFT));
   this.editors.right.ace.setTheme(getTheme(this, C.EDITOR_RIGHT));
+  this.editors.left.ace.$blockScrolling = Infinity;
```
Comment (JackuB) [inline src/index.js]: Could you also add a code comment to these lines so it's clear what they are doing - possibly also linking to some Ace docs or something

File: .npmignore (hunk 2 above)
Comment (JackuB) [reply .npmignore]: You can see them being introduced here https://github.com/ace-diff/ace-diff/blob/master/webpack.config.js#L12-L16

File: .gitignore (hunk 1 above)
Comment (sufius) [reply .gitignore]: you are right, thank you

File: .npmignore (hunk 2 above)
Comment (sufius) [reply .npmignore]: you are right, thank you

File: src/index.js (hunk 3 above)
Comment (sufius) [reply src/index.js]: done

**Comment by JackuB** [comment]
@sufius thanks a lot! Last thing: could you please just squash commits to a single one with a message like `fix(block_scrolling): avoid deprecation message` - this way, it will get correctly added and published here https://github.com/ace-diff/ace-diff/releases just like the other ones.

For example this commit [`feat(css): add dark mode CSS`](https://github.com/ace-diff/ace-diff/commit/b2953e5) produced this release https://github.com/ace-diff/ace-diff/releases/tag/v2.2.0

If you need help with that, just write here I can help you.

**Comment by sufius** [comment]
hi, so if I got it correct, i should, by for example rebasing, change the name of each of previously pushed commit message? If yes, is there a easier way too? If no, how exactly I should do this? Thanks

**Comment by JackuB** [comment]
Resolving this as part of https://github.com/ace-diff/ace-diff/pull/69

**Pull request closed without merging**